packages/jest-util/src/is_interative.js

```diff
@@ -1,3 +1,3 @@
 import isCI from 'is-ci';
 
-export default process.stdout.isTTY && !isCI;
+export default process.stdout.isTTY && process.env.TERM !== 'dumb' && !isCI;
```

Comment:
Feels like there should exist some module which has this logic already

Reply:
I had the same feeling, but I couldn't find one that specifically checks for a dumb terminal without doing other stuff. For example [supports-color](https://github.com/chalk/supports-color) takes into account if a terminal is dumb but it's coupled with color related features that we don't want to check in this case.